### ISSUE TITLE
[EN][Quest][Netherstorm] Various quests in Ethereum Staging Grounds

### ISSUE THREAD

**Issue by Illusionist87** [Atlantiss/BugTracker#4912]
http://wowwiki.wikia.com/wiki/Ethereum_Prison

Ethereum Prisons that should be all over Ethereum Staging Grounds are not there. The bases are there but the clickable pink orb or the actual prison is missing. This makes various quests impossible to complete. 

For example:
Ethereum Secrets - http://www.wowhead.com/quest=10971
Captain Tyralius - http://www.wowhead.com/quest=10422

It might be the same bug as with other "save from the prison quests" (like the "HELP!" quest in Nagrand) where the prison despawns after being used by a player and doesn't reappear until a server crash / restart / shut-down.

**Comment by Karathe** [comment]
This problem seems to keep occurring.

**Comment by tomatosz** [comment]
Bump, tried Captain Tyralius after server restart but still not working.

**Comment by Damfee** [comment]
That's odd. I've done Captain Tyralis after a server restart maybe 4 days ago? Then I did the entire chain after that all the way up to the 2 Mana-Tombs prison. 

Maybe someone did it before you?

**Comment by tomatosz** [comment]
You're right, someone must've done it in the 10 min before I got there. Finished it now after another restart.

Op 16 okt. 2016 om 17:47 heeft "Damfee" <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


That's odd. I've done Captain Tyralis after a server restart maybe 4 days ago? Then I did the entire chain after that all the way up to the 2 Mana-Tombs prison.

Maybe someone did it before you?

-

**Comment by aendry** [comment]
this is still broken. the server has just restarted and I tried to do ethereum secrets but no luck

**Comment by Noviush** [comment]
Both quest still bugged. How long takes to repair TBC quests? This two are bugged over a year...
